pyproject.toml

```diff
@@ -93,3 +94,6 @@ module = [
     "two_factor.*",
 ]
 ignore_missing_imports = true
+
+[tool.django-stubs]
+django_settings_module = "zproject.test_settings"
```

Comment:
I think we want to type-check against `zproject.settings`, not `zproject.test_settings`, which might have narrower types.